programs/marinade-finance/src/instructions/admin/config_marinade.rs

```diff
@@ -156,6 +163,51 @@ impl<'info> ConfigMarinade<'info> {
                 None
             };
 
+        let withdraw_stake_account_enabled_change =
+            if let Some(withdraw_stake_account_enabled) = withdraw_stake_account_enabled {
+                let old = self.state.withdraw_stake_account_enabled;
+                self.state.withdraw_stake_account_enabled =withdraw_stake_account_enabled;
+                Some(BoolValueChange {
+                    old,
+                    new: withdraw_stake_account_enabled,
+                })
+            } else {
+                None
+            };
+
+        let delayed_unstake_fee_change = if let Some(delayed_unstake_fee) = delayed_unstake_fee {
+            require_lte!(
+                delayed_unstake_fee.bp_cents,
+                State::MAX_DELAYED_UNSTAKE_FEE.bp_cents,
+                MarinadeError::RewardsFeeIsTooHigh
+            );
+            let old = self.state.delayed_unstake_fee;
+            self.state.delayed_unstake_fee = delayed_unstake_fee;
+            Some(FeeCentsValueChange {
+                old,
+                new: delayed_unstake_fee,
+            })
+        } else {
+            None
+        };
+
+        let withdraw_stake_account_fee_change =
+            if let Some(withdraw_stake_account_fee) = withdraw_stake_account_fee {
```

Comment:
Please use Ord trait

Reply:
explained in other comment, required to reduce 100K from .so file

Reply:
fixed